beacon-chain/blockchain/receive_block.go

```diff
@@ -86,7 +86,8 @@ func (s *Service) ReceiveBlockBatch(ctx context.Context, blocks []block.SignedBe
 
 	for i, b := range blocks {
 		blockCopy := b.Copy()
-		if err = s.handleBlockAfterBatchVerify(ctx, blockCopy, blkRoots[i], fCheckpoints[i], jCheckpoints[i]); err != nil {
+		// TODO(10261) check optimistic status
+		if err = s.handleBlockAfterBatchVerify(ctx, blockCopy, blkRoots[i], fCheckpoints[i], jCheckpoints[i], false /*optimistic status*/); err != nil {
```

Comment:
We do care, here it is false because we haven't implemented the right check. That's why the TODO and the issue linked. We will implement that check in a future PR in kiln first